buildifier/buildifier.go

```diff
@@ -200,6 +203,10 @@ func main() {
 	}
 
 	diff = differ.Find()
+	if *diffProgram != "" {
+		diff.Cmd = *diffProgram
```

Comment:
Unfortunately, it seems like it does not allow me to specify arguments with the command. E.g.:

    BUILDIFIER_DIFF="diff -u" buildifier [...]

works, but

    buildifier -diff="diff -u" [...]

gives me

    ./BUILD:
    --: diff -u: command not found
    exit status 127

Or am I doing something wrong?

(I could not find an obvious reason why it works via the environment; I'm not too skilled in Go.)

Reply:
I think you must be doing something wrong.  Using -diff_command="diff -u" works for me.

Reply:
Apparently I was doing something wrong indeed. It's working fine with your commit and

```
buildifier(
    name = "buildifier_diff",
    diff_command = "diff -u",
    mode = "diff",
    verbose = True,
)
```

:+1: